ext/npm-extension.js

```diff
@@ -393,11 +393,14 @@ exports.addExtension = function addNpmExtension(System){
 			if ((statusCode === 404 || statusCode === 0) &&
 				utils.moduleName.isBareIdentifier(load.name) &&
 				!utils.pkg.isRoot(loader, load.metadata.npmPackage)) {
-				var newError = new Error([
-					"Could not load '" + load.name + "'",
+				var errorMsg = ["Could not load '" + load.name + "'",
 					"Is this an npm module not saved in your package.json?"
-				].join("\n"));
+				].join("\n");
+				var newError = new Error();
```

Comment:
I think you want to pass errorMsg into the constructor, otherwise it will not be part of the current `.stack`, so prepending won't add this part.